package.json

```diff
@@ -46,7 +46,7 @@
   },
   "grafana": {
     "whatsNewUrl": "https://grafana.com/docs/grafana/latest/guides/whats-new-in-v7-3/",
-    "releaseNotesUrl": "https://community.grafana.com/t/release-notes-v7-3-x/37993"
+    "releaseNotesUrl": "https://grafana.com/docs/grafana/latest/release-notes/"
```

Comment:
Shouldn't this be something like: `https://grafana.com/docs/grafana/latest/release-notes/release-notes-7-3-4/`?

Reply:
not sure, was thinking it could go to the index page so we do not have to update this every version bump. But we could change this in the publish code as well now that the URL is deterministic and only includes the version